samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -65,23 +73,124 @@ const StorybookDecorator = (story, config = {}) => {
 		classes.debug = true;
 	}
 
+	// beginning of custom theme code
+	const request = new LS2Request();
+	const [context, setContext] = useState(customColorsContext);
+
+	const defaultKeyThemeValue = JSON.stringify({
+		"version": "0.1",
+		"activeTheme": "defaultTheme",
+		"backgroundColor": "#000000",
+		"componentBackgroundColor": "#7D848C",
```

Comment:
I don't have any specific idea but it looks better to gather static default values in one place so that we would not miss some values when design updates.

Reply:
Since I updated sampler to only read settings' values, this is not needed anymore.